gapis/api/reference.go

```diff
@@ -45,3 +47,9 @@ type NilReference struct{}
 func (NilReference) RefID() RefID {
 	return NilRefID
 }
+
+func init() {
+	// Don't compare RefIDs.
+	compare.Register(func(c compare.Comparator, a, b RefID) {
```

Comment:
No need to change now, but we should probably avoid creating these global comparison functions - I've found they hid differences that you might want to find under different circumstances. The compare package needs a little bit of work to expose this, but we should probably be passing the custom rules down with each compare.

Reply:
Agreed.